controller.py

```diff
@@ -0,0 +1,43 @@
+from typing import List, Dict
+
+from db.iclient_db import IClientDB
+from entities.card import Card
+from entities.card_proprties import CardProperties
+from entities.status import Status
+
+from flask import Flask, request
+from consts import *
+
+
+CARDS_SERIALIZED = Dict[str, List[Dict[str, str]]]
+
+
+class Controller:
+    def __init__(self, client_db: IClientDB):
+        self.app = Flask(__name__)
+        self.client_db = client_db
+        self.setup_routes()
```

Comment:
Isn't this action a bit "too much" for const'? 
Consider moving it to `__enter__`